earth_enterprise/src/common/khCache.h

```diff
@@ -260,7 +304,9 @@ class khCache {
   void Prune(void) {
     CheckListInvariant();
     Item *item = tail;
-    while (item && (map.size() > targetMax)) {
+    while (item && (
```

Comment:
Minor: you could make this a bit clearer by moving the boolean logic here into a function, something like
```
while (item && cacheIsTooBig())
```
Then you can make the function as expressive as you like:
```
bool cacheIsTooBig() {
  if (limitCacheMemory) return cacheMemoryUse > maxCacheMemory;
  else return map.size() > targetMax;
}
```
or something like that.